Jenkinsfile

```diff
@@ -5,12 +5,29 @@ elifePipeline {
         commit = elifeGitRevision()
     }
 
-    stage 'Project tests', {
-        lock('journal--ci') {
-            builderDeployRevision 'journal--ci', commit
-            builderProjectTests 'journal--ci', '/srv/journal', ['/srv/journal/build/phpunit/*.xml', '/srv/journal/build/behat/*.xml'], ['smoke', 'project']
-        }
-    }
+    elifeOnNode(
+        {
+            stage 'Build images', {
+                checkout scm
+                sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml build"
+            }
+
+            stage 'Project tests', {
+                sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml up -d"
+                dockerComposeProjectTestsParallel('journal', commit, [
+                    'phpunit': '/srv/journal/build/phpunit/*.xml',
+                    'behat': '/srv/journal/build/behat/*.xml'
+                ])
+
+                try {
+                    sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml up -d cli fpm web"
+                } finally {
+                    sh 'docker-compose -f docker-compose.yml -f docker-compose.ci.yml down'
```

Comment:
Probably need to add `--volumes`, though the problem will actually be the equivalent in `dockerComposeProjectTestsParallel`?

Reply:
It seems reasonable to always tear down volumes during different runs of project tests, so I'll add it there.

Reply:
at the moment it's `dockerComposeSmokeTests` that runs `docker-compose down`, but nothing prevents `dockerComposeProjectTests*` steps from doing that too.